templates/disk-uploader/manifests/disk-uploader.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.43.0"
+    tekton.dev/categories: Automation
+    tekton.dev/tags: kubevirt, containerdisks
+    tekton.dev/displayName: "{{ display_name_provider }} {{ nice_name }}"
+    tekton.dev/platforms: "linux/amd64"
+    artifacthub.io/maintainers: |
+      - name: {{ maintainer_name }}
+        email: {{ maintainer_email }}
+    artifacthub.io/provider: {{ provider }}
+    artifacthub.io/recommendations: |
+      - url: {{ recommendation_url }}
+    artifacthub.io/category: integration-delivery
+  labels:
+    app.kubernetes.io/version: {{ catalog_version }}
+  name: {{ task_name }}
+spec:
+  description: >-
+    Automates the extraction of disk and uploads it to a container registry,
+    to be used in multiple Kubernetes clusters.
+  params:
+  - name: EXPORT_SOURCE_KIND
+    description: The name of the export source kind
+    type: string
+  - name: EXPORT_SOURCE_NAME
+    description: The name of the export source
+    type: string
+  - name: VOLUME_NAME
+    description: The volume name (If source kind is PVC, then volume name is equal to source name)
+    type: string
+  - name: IMAGE_DESTINATION
+    description: Destination of the image in container registry
+    type: string
+  - name: PUSH_TIMEOUT
+    description: ContainerDisk push timeout in minutes
+    type: string
+  steps:
+  - name: disk-uploader-step
+    image: "{{ main_image }}:{{ version }}"
+    env:
+    - name: ACCESS_KEY_ID
+      valueFrom:
+        secretKeyRef:
+          name: disk-uploader-credentials
+          key: accessKeyId
+    - name: SECRET_KEY
+      valueFrom:
+        secretKeyRef:
+          name: disk-uploader-credentials
+          key: secretKey
+    - name: POD_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.name
+    command: ["/usr/local/bin/disk-uploader"]
+    args:
+    - "--export-source-kind"
```

Comment:
When I tried to run the task, it ended with error:
```
Usage: disk-uploader [--export-source-kind EXPORT-SOURCE-KIND] [--export-source-namespace EXPORT-SOURCE-NAMESPACE] [--export-source-name EXPORT-SOURCE-NAME] [--volume-name VOLUME-NAME] [--image-destination IMAGE-DESTINATION] [--push-timeout PUSH-TIMEOUT] [--debug]
error: unknown argument --volumename
```

Reply:
Resolved. Thanks!